### PR TITLE
2.x Doc updates for pachctl deploy/helm - draft

### PR DESCRIPTION
**Pull request by tybritten**
Remove the rest of the `pachctl deploy` references and include a helm values.yaml reference

### CODE REVIEW

File: doc/mkdocs-master.yml [hunk 1]
```diff
@@ -236,6 +236,7 @@ nav:
         - Community Edition Limits: reference/scaling_limits.md
         - Pipeline Specification: reference/pipeline_spec.md
         - Pachyderm Config Specification: reference/config_spec.md
+        - Pachyderm Helm Chart Vales Specification: reference/helm_values.md
```
Comment (nadegepepin) [inline doc/mkdocs-master.yml:239]: Values

**Comment by tybritten** [comment]
@chainlink can you look at the helm reference section and let me know what needs to be tweaked?

**Comment by chainlink** [comment]
Will do!

**Review by nadegepepin — APPROVED** [review]
LGTM

**Pull request merged**